### PR TITLE
Ft blog integration #158174600

### PR DESCRIPTION
**Pull request by mutebironald**
## Status
**READY**

## Migrations
YES 

## Description
A basic blog that allows logged in users to create categories, write blog posts and share blog posts.
- Given users want to contribute knowledge 
- When they login to their account
- Then they should be able to create categories, write blog posts and share those posts publicly

## Steps to Test 
Clone the repository and run the following commands

- python manage.py test
- python manage.py ensuretriggers
- python manage.py makemigrations
- python manage.py migrate

## Impacted Areas in Application
Components of the application that this PR will affect:
Settings
hc

## What are the relevant pivotal tracker stories?
[#158174600](url)

screen shot 2018-07-17 at 18 58 54
screen shot 2018-07-17 at 18 59 14

### CODE REVIEW

**Comment by EdwinKato** [comment]
@mutebironald Use the assign widget for the 2 other reviewers..

**Review by magicmarie — DISMISSED** [review]
LGTM.

**Comment by EdwinKato** [comment]
@mutebironald Add screenshots

**Comment by mutebironald** [comment]
@EdwinKato Added the screenshots

**Comment by archbishop12** [comment]
You should fix the flask8 issues.

This message keeps showing up even after one creates a post otherwise the code looks good and is running.
screen shot 2018-07-18 at 13 50 14

File: .travis.yml [hunk 1]
```diff
@@ -22,9 +22,9 @@ notifications:
...
-  skip_cleanup: true
-  app: health-checks-aces-staging
-  on:
-    repo: andela/hc-aces-kla
-    branch: develop
\ No newline at end of file
+    secure: kg/YJsYRXxT3MjtdSjgOpNJKKP5LGy8SQE+fYqyMphp/trpEkVAezYmII+BdvAVScqh4pwAvqkcttYpFCUgfzxyy0hhOcVQaDbusUF5bbs5ChAbwR4ftIIDUr82Gd6xEaWIy0SQWUel4QBeezXACI70hN0G/N/znPojGmxzZdcsZm3d90yVF+AraXW1Z1/BLmFlx+LAhbzlg8888QfM5m5F7bKlcqC5koKxWcZc9SrZugIQGhR/MHoBxvK9OpkHBgAh4iOxqxYA90A4Xj2PNBXV2aVSpNDw3RVEW6p7w3pc0wSmiOqzUrwU2ZzvYUVXKTLfGfxokv3Kj9kqGDKqOIL8xzaMP1cU1JkEZtim6xanYokVU4ySVqcODQ57zA55zLK+uc1G1b7BT/SDrGyindpuC5Px3fteoydMabczeQxTi3vxPAN+rjzxWD2JeYUX5hSlk9wKhfh8OyU4ve/F/UThqoVvPcZxbRG4cmMimnsx7g/sFhJDgBzG1/zUjLb4vyYL8M1lIacGtCF2lvACP5UvTak4UoDKHKNDzIkxaAQfd2YzX12bWrYVyMDeM3m4zjm9+GvRWzk0/cPd5cNleraErYVqeCYBPJJXFF/1ErYXPZozOLJh7NYqvtzsrp64QyedH0AOHcdOCly5FetQ4kX3tu6Bj2TSMcV5TQLsnpq0=
```
Comment (EdwinKato) [inline .travis.yml]: Is there any reason this file is changing in this PR? Consult with @matthewacha and @magicmarie on this. This change could cause travis to fail in future builds

**Review by EdwinKato — CHANGES_REQUESTED** [review]
Resolve flake8 issues before I can do a full review

File: .travis.yml (hunk 1 above)
Comment (mutebironald) [reply .travis.yml]: @EdwinKato Let me fix this

Comment (mutebironald) [reply .travis.yml]: @EdwinKato resolved

**Comment by EdwinKato** [comment]
@mutebironald Resolve conflicts. It also looks like you have a few issues in your migrations. They do not work locally on my machine. Kindly confirm this with the other reviewers

**Comment by mutebironald** [comment]
@EdwinKato, I have fixed the naming and migration issues.

**Comment by EdwinKato** [comment]
Hi @mutebironald any chance you could close this pull request? It shows up in some of my lists

**Comment by mutebironald** [comment]
Hi @EdwinKato, let me close it👍. Cheers.

**Pull request closed without merging**